.github/workflows/update-template.yml

```diff
@@ -68,6 +68,7 @@ jobs:
     runs-on: ubuntu-latest
     outputs:
       error: ${{ steps.run-update-template.outputs.error }}
+      status: ${{ steps.run-update-template.outputs.status }}
```

Comment:
@AmyJZhao , you built a reusable "needs-more-verification" workflow, please use it here as well to decide if a template needs an additional verification step or not

Reply:
I think this change needs to go here? https://github.com/adobe/aio-template-submission/pull/273/files#diff-d7a677c9ed6b55c54b4c54d6de22d4c5832d355a6c9a22ffd98de08932aa8074L71